src/common/style/vanilla/navigation.scss

```diff
@@ -60,15 +73,55 @@ $color-navigation-background: #252525;
   // right aligned sign-in navigation
   .p-navigation__links--right {
     @extend .p-navigation__links;
+    margin-right: 20px;
+
+    i {
```

Comment:
If possible this could be more specific - maybe a full icon class name?

Reply:
done, thanks for the help on IRC